travvy/src/components/LogIn.vue

```diff
@@ -5,11 +5,11 @@
         <h2>Log In</h2>
         <form>
         <!-- this form takes the sign up input from users - binds to the user form --> 
-          <label><strong>Email: </strong></label>
+          <label><strong>Email: </strong></label><br><br>
```

Comment:
Thanks for doing this, I am not sure why it already looked like there were line breaks when I checked on my local